### PR TITLE
Add Emancipation status effect

### PR DESCRIPTION
**Pull request by Pyrofab**
This PR adds the emancipation status effect as described in the [relevant project](https://github.com/Ladysnake/Requiem/projects/7#card-51886099).

### CODE REVIEW

File: src/main/java/ladysnake/requiem/common/remnant/BasePossessionHandlers.java [hunk 1]
```diff
@@ -82,6 +83,12 @@ public static void register() {
             }
             return PossessionStartCallback.Result.PASS;
         });
+        PossessionStartCallback.EVENT.register(Requiem.id("emancipation"), (target, possessor, simulate) -> {
+            if (possessor.hasStatusEffect(RequiemStatusEffects.EMANCIPATION)) {
+                return PossessionStartCallback.Result.ALLOW;
```
Comment (SekoiaTree) [inline src/main/java/ladysnake/requiem/common/remnant/BasePossessionHandlers.java:89]: Wouldn't this allow you to possess anything, without any other addons being able to override this? Possibly unintended/could cause issues

**Review by SekoiaTree — COMMENTED** [review]
Seems pretty good just one thing to request rq. Very nice cleanup changes too :+1:

File: src/main/java/ladysnake/requiem/common/remnant/BasePossessionHandlers.java (hunk 1 above)
Comment (Pyrofab) [reply src/main/java/ladysnake/requiem/common/remnant/BasePossessionHandlers.java:89]: Correct ! I added a TODO to remind ourselves to fix this after soul vessels are done.

**Review by SekoiaTree — APPROVED** [review]
all good!

**Pull request merged**